apps/colt-steele/seed_data/01.sql

```diff
@@ -160,6 +160,7 @@ CREATE TABLE `Price` (
 CREATE TABLE `Product` (
     `id` VARCHAR(191) NOT NULL,
     `name` VARCHAR(191) NOT NULL,
+    `productType` VARCHAR(191) NOT NULL,
```

Comment:
a few of these `apps/<app-name>/seed_data/01.sql` files have `NOT NULL` on the `productType` which doesn't match the prisma schema